src/components/Notification.stories.tsx

```diff
@@ -0,0 +1,60 @@
+import { ComponentStory } from '@storybook/react';
+import { Notification } from './Notification';
+
+export default {
+  title: 'Components/Notification',
+  component: Notification,
+  parameters: {
```

Comment:
Storybook confusingly wraps the `message` value in quotes because its type is `ReactNode` and not `string`:

![image](https://user-images.githubusercontent.com/1060248/227987626-2d01773b-88df-4c4a-ab10-1863bc825c95.png)

Its behavior can be fixed by specifying an arg type:

```suggestion
  argTypes: {
    message: { control: 'text' }
  },
  parameters: {
```